src/js/stores/AppVersionsStore.js

```diff
@@ -25,22 +25,22 @@ AppDispatcher.register(function (action) {
   switch (action.actionType) {
     case AppVersionsEvents.REQUEST_VERSION_TIMESTAMPS:
       AppVersionsStore.resetOnAppChange(action.appId);
-      AppVersionsStore.availableAppVersions = action.data;
+      AppVersionsStore.availableAppVersions = action.data.body;
```

Comment:
For now this is OK. I will start another PR, after this is merged to bring the ```.body``` into the Actions.
So we don't have so much repetition here and can use ```action.data``` even in the ERRORs.
```
    .success(function (response) {
      AppDispatcher.dispatch({
        actionType: AppsEvents.REQUEST_APPS,
        data: response.body
      });
    })
```

I will take care of this..